testutil/garbage/garbage.go

```diff
@@ -0,0 +1,316 @@
+package garbage
+
+import (
+	"math"
+	"math/rand"
```

Comment:
I would suggest to use `mathrand "math/rand"` for two reasons:

1) wherever it is used, it's clear it's math/rand, not crypto/rand
2) you can then use `rand` as a parameter name without shadowing, which can avoid bugs like unintentionally using the _global_ `rand.Func` rather than the `rand.Method` on the parameter of type `rand.Rand`.

Reply:
:thumbsup: good suggstion